includes/class-custom-taxonomies.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * The core plugin class file
+ *
+ * Defines the functions necessary to register our custom taxonomies with
+ * WordPress.
+ *
+ * @link       http://code.tutsplus.com/series/the-beginners-guide-to-wordpress-taxonomies--cms-706
+ * @since      1.0.1
+ *
+ * @package    WSUWP_Plugin_iDonate_Custom_Taxonomies
+ * @author     Blair Lierman <blair.lierman@wsu.edu>
+ */
+class WSUWP_Plugin_iDonate_Custom_Taxonomies {
+
+	/**
+	 * Initializes the plugin by registering the hooks necessary
+	 * for creating our custom taxonomies within WordPress.
+	 *
+	 * @since    1.0.0
+	 */
+	public function init() {
+
+		add_action( 'init', array( $this, 'init_taxonomies' ) );
+
+	}
+
+	public function init_taxonomies() {
+		$taxonomies = array();
+
+		$taxonomies[] = array(
+			'name' => 'priorities',
```

Comment:
Similar to the post type slug, we should prefix the taxonomy slugs. There is a higher chance of conflict here because of other university type taxonomies that may be in use.

* `idonate_priorities`, etc...